src/components/roundCornerBox.js

```diff
@@ -0,0 +1,64 @@
+const RoundedBoxGeometry = require("three-rounded-box")(THREE);
+
+AFRAME.registerComponent("roundbox", {
+  schema: {
+    borderRadius: { type: "number", default: 0.15 },
+    opacity: { type: "number", default: 1 },
+  },
+  init: function () {
+    this.ascend = true;
+    const opacity = this.data.opacity;
```

Comment:
Can we do object destruction here for clarity? 

`const { opacity, borderRadius } = this.data;`

Reply:
Resolved